arches_querysets/utils/models.py

```diff
@@ -26,32 +26,39 @@
 from arches.app.models.models import ResourceInstance, TileModel
 
 from arches_querysets.fields import (
+    CardinalityNConceptListField,
+    CardinalityNJSONField,
     CardinalityNResourceInstanceField,
-    CardinalityNStringField,
+    CardinalityNResourceInstanceListField,
+    CardinalityNLocalizedStringField,
+    CardinalityNTextField,
+    CardinalityNUUIDField,
+    ConceptListField,
+    LocalizedStringField,
     ResourceInstanceField,
     ResourceInstanceListField,
-    StringField,
 )
 
 
 DATATYPES_NEEDING_KEY_TEXT_TRANSFORM = {
     "non-localized-string",
     "date",
     "concept",
+    "concept-list",
     "node-value",
 }
-DATATYPES_NEEDING_CAST = {"boolean", "date", "number"}
+DATATYPES_NEEDING_CAST = {"boolean", "concept-list", "date", "number"}
```

Comment:
The long-term vision is for this to be expressed on the datatype classes themselves, but we're putting one foot in front of the other while demonstrating viability.